pkg/controller/migplan/migplan_controller.go

```diff
@@ -157,9 +158,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 var _ reconcile.Reconciler = &ReconcileMigPlan{}
 
+type Identities struct {
+	SrcIdentity  *migauth.Identity
+	DestIdentity *migauth.Identity
```

Comment:
All of the fields of a struct are properties of (describe) the _type_ so each filed containing the name (form of) of the _type_ is redundant.  This reads like: `Identities.dstIdentity`.

Recommend.
```
type Identities struct {
    source *migauth.Identity
    destination *migauth.Identity
}
```

Reply:
Done.